sandbox/src/ActivityMapExtension.js

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect, useRef } from "react";
```

Comment:
We can remove this file unless we want a page that uses the extension (which I don't think we do atm).